dbschema/partnership.esdl

```diff
@@ -0,0 +1,38 @@
+module default {
+  type Partnership extending Project::Child {
+    # todo: implement Changeset type?
+    # declare readonly scope: ScopedRole[];
+
+    agreementStatus: Partnership::PartnershipAgreementStatus;
+    mou: File; 
+    mouStatus: Partnership::PartnershipAgreementStatus;
+    mouStart := .mouStartOverride ?? .project.mouStart;
+    mouEnd := .mouEndOverride ?? .project.mouEnd;
+    mouStartOverride: cal::local_date;
+    mouEndOverride: cal::local_date;
+    agreement: File;
+    required partner: Partner;
+    required organization: Organization;
+    multi types: Partner::Type;
+    multi financialReportingTypes: Partnership::FinancialReportingType;
+    primary: bool {
+      default := false;
+    }
+  }
+}
+
+module Partnership {
+  scalar type PartnershipAgreementStatus extending enum<
+    NotAttached,
+    AwaitingSignature,
+    Signed
+  >;
+
+  #Confirm this type should be removed from Partner.esdl and when it should be removed
+  scalar type FinancialReportingType extending enum<
+    Funded,
+    FieldEngaged,
+    Hybrid
+  >;
```

Comment:
You can keep this here and remove it from `Partner` now.  Just make sure you also change any references to it to `Partner::FinancialReportingType`.